src/pages/SignUp/SignupPage.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useRegisterUserMutation } from '../../services/authApi';
+import SignupForm from '../../components/Forms/SignUpForm';
+import { ROUTES } from '../../constants/routerPaths';
+import SharedTypography from '../../components/shared/Text/SharedText';
+import { AUTH_ERRORS } from '../../constants/auth.constants';
+
+export default function SignupPage() {
+  const [error, setError] = useState('');
+  const navigate = useNavigate();
+  const [registerUser, { isLoading }] = useRegisterUserMutation();
```

Comment:
You are using react-query, so you can destruct the `error` object from `useRegisterUserMutation()` (instead of managing another state in the component (`error`)